core/install.js

```diff
@@ -14,7 +14,7 @@ var npm = require('npm'),
 	fs = require('fs'),
 	load = deasync(npm.load);
 
-load({loglevel: "silent"});
+load({loglevel: 'silent'});
 var inst = deasync(npm.commands.install),
 upd = deasync(npm.commands.update),
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 space characters but found 0.](https://www.codacy.com/app/theprojectshop/Kassy/file/2554993052/issues/source?bid=3282987&fileBranchId=3284644#l19)